tutorials/scripting/cpp/build_system/scons.rst

```diff
@@ -0,0 +1,72 @@
+.. _doc_godot_cpp_build_system:
```

Comment:
I would add a `seealso::` at the beginning of this page that points to https://docs.godotengine.org/en/stable/contributing/development/compiling/introduction_to_the_buildsystem.html, for users looking to compile Godot itself. It's likely this page will come up in search results for users looking to compile Godot, rather than godot-cpp.

Reply:
That's a good point. I added one in this page, and an identical one on the cmake page (since 100% people are going to be searching for "CMake" in the repo as well).